lochness/__init__.py

```diff
@@ -43,6 +43,48 @@
 ])
 
 
+class Subject(object):
+    def __init__(self, active, phoenix_study, phoenix_id, consent, beiwe,
+                 icognition, saliva, xnat, redcap, dropbox,
+                 box, mediaflux, mindlamp, daris, rpms,
+                 general, protected, metadata_file):
+        self.active = active
+        self.study = phoenix_study
+        self.id = phoenix_id
+        self.consent = consent
+        self.beiwe = beiwe
+        self.icognition = icognition
+        self.saliva = saliva
+        self.xnat = xnat
+        self.redcap = redcap
+        self.dropbox = dropbox
+        self.box = box
+        self.mediaflux = mediaflux
+        self.mindlamp = mindlamp
+        self.daris = daris
+        self.rpms = rpms
+        self.general_folder = general
+        self.protected_folder = protected
+        self.metadata_csv = metadata_file
+
+    def asdict(self):
+        '''emulating collection._asdict()'''
+        return {'active': self.active, 'study': self.study,
+                'id': self.id, 'consent': self.consent,
+                'beiwe': self.beiwe, 'icognition': self.icognition,
+                'saliva': self.saliva, 'xnat': self.xnat,
+                'redcap': self.redcap, 'dropbox': self.dropbox,
+                'box': self.box, 'mediaflux': self.mediaflux,
+                'mindlamp': self.mindlamp, 'daris': self.daris,
+                'rpms': self.rpms, 'general_folder': self.general_folder,
+                'protected_folder': self.protected_folder,
+                'metadata_csv': self.metadata_csv}
+        
+    def to_bids(self):
```

Comment:
`to_bids` is ran when `BIDS: True` is present in the `config.yml`

Reply:
https://github.com/PREDICT-DPACC/lochness/blob/master/docs/source/mediaflux.md#configuration-file

1. Should I write `BIDS: True` in the mediaflux block or in the outer block?

2. Is `BIDS` case senstive?

Reply:
1.  Chris' comment below suggests us to have the BIDS true as the default option. Don't need to add BIDS: True. (I have not worked on the tree module yet)
2. yes

Reply:
> No need to add

Hi @kcho , this does not appear right. I get an error albeit a warning:

> 2021-06-17 14:21:36,836 - 84880 - lochness - WARNING - 'BIDS'

If I do not write `BIDS` in configuration.

Reply:
It's surprising that lochness throws a mere warning other than raising an error for these mistakes!

Reply:
This is the line that catches all exceptions as warnings:
https://github.com/PREDICT-DPACC/lochness/blob/kcho/BIDS/lochness/__init__.py#L385

It's probably not the best way but I don't think we have time to deal with it! For now, I am keeping `BIDS` parameter in the config.

Please confirm that `lochness_create_template.py` does that too.